synchronization-service/src/main.py

```diff
@@ -31,15 +32,10 @@ async def send_unsychronized_votes(votes) -> requests.Response:
     my_private_key = requests.get('http://web/temporary_key_location/private_key.txt').text
 
     for vote in votes:
-        new_vote = {
-            'token' : vote['vote']['token'],
-            'election_id' : vote['vote']['election_id'],
-            'party_id' : int(vote['vote']['party_id']),
-            'candidates_ids': vote['vote']['candidates'],
```

Comment:
ma vote parameter `candidates`? (rovnako o par riadkov nizsie)

Reply:
no, nemá. Toto sú práveže odstránené riadky. To `candidates` bolo ešte v predošlom formáte hlasu v databáze na G. Teraz sú tieto "konvertovacie" riadky preč a validuje sa to riadkom 35 podľa objektu Vote z toho datamodels.